### DISCUSSION TITLE
Export task with json file

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4044] [Q&A]
When we export task, the 'project_id' field is not shown in the task.json file. Can i know how can the 'project_id" field be added in exported task.json file?
![task](https://user-images.githubusercontent.com/16021947/146504844-59b9a59e-b75a-4723-939c-c7ab93fb5897.jpg)

**Comment by bsekachev** [comment]
@azhavoro Could you comment on?

I think if we export project id here, importing it on another instance in general leads to an error because such project does not exist / we do not have access to it. Recently we added a feature to backup/restore the whole project.

**Reply by keshva23** [reply]
@azhavoro Ok, Thank you.
